functions.php

```diff
@@ -127,8 +127,9 @@ function twentyten_judge_child_wp_trim_excerpt( $text = '' ) {
 		 * @param string $more_string The string shown within the more link.
 		 */
 		$excerpt_more = apply_filters( 'excerpt_more', ' ' . '[&hellip;]' );
-		$text = wp_trim_words( $text, $excerpt_length, $excerpt_more );
+		$text         = wp_trim_words( $text, $excerpt_length, $excerpt_more );
```

Comment:
Oh I see, what you are trying to do. Do mid-line alignment only for associative arrays. These code lines have nothing to do with each other and thus shouldn't be aligned this way.

https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#indentation